core/lib/server/pub.ts

```diff
@@ -274,7 +272,6 @@ export const _deprecated_getPubs = async (
 					.innerJoin("PubsInStages", "pubs.id", "PubsInStages.pubId")
```

Comment:
it's not in the PR so i can't comment on the lines directly, but there's an unused param related to children in this file a little above here:
https://github.com/pubpub/v7/blob/d53bc8e2e80127e0e3f6a3f8035c00a082efdca4/core/lib/server/pub.ts#L232-L238
https://github.com/pubpub/v7/blob/d53bc8e2e80127e0e3f6a3f8035c00a082efdca4/core/lib/server/pub.ts#L246

Reply:
Another nice catch, thank you!